src/patternfly/components/Nav/nav.scss

```diff
@@ -230,12 +146,52 @@
 
-  // Base styling
+
+  // List toggle
+  --pf-c-nav__toggle--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-nav__toggle--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-nav__toggle--FontSize: var(--pf-global--icon--FontSize--md);
+  --pf-c-nav__toggle--Transition: var(--pf-global--TransitionDuration);
+
+  // Divider
+  --pf-c-nav__c-divider--MarginTop: var(--pf-global--spacer--sm);
+  --pf-c-nav__c-divider--MarginBottom: var(--pf-global--spacer--sm);
+  --pf-c-nav--c-divider--PaddingRight: 0;
+  --pf-c-nav--c-divider--PaddingLeft: 0;
+
+  @media screen and (min-width: $pf-global--breakpoint--md) {
+    --pf-c-nav__list__link--PaddingRight: var(--pf-c-nav__list__link--md--PaddingRight);
+    --pf-c-nav__list__link--PaddingLeft: var(--pf-c-nav__list__link--md--PaddingLeft);
+    --pf-c-nav__section-title--PaddingRight: var(--pf-c-nav__section-title--md--PaddingRight);
+    --pf-c-nav__section-title--PaddingLeft: var(--pf-c-nav__section-title--md--PaddingLeft);
+    --pf-c-nav__horizontal-list__link--PaddingTop: var(--pf-c-nav__horizontal-list__link--md--PaddingTop);
+    --pf-c-nav__subnav--PaddingLeft: var(--pf-c-nav__subnav--md--PaddingLeft);
+  }
+
+  @media screen and (min-width: $pf-global--breakpoint--lg) {
+    --pf-c-nav__horizontal-list__link--PaddingBottom: var(--pf-c-nav__horizontal-list__link--lg--PaddingBottom);
+  }
+
   position: relative;
 
-  .pf-c-nav__scroll-button {
-    display: none;
-    visibility: hidden;
+  &.pf-m-dark {
+    --pf-c-nav__link--BackgroundColor: var(--pf-c-nav--m-dark__link--BackgroundColor);
+    --pf-c-nav__link--Color: var(--pf-c-nav--m-dark__link--Color);
+    --pf-c-nav__link--hover--Color: var(--pf-c-nav--m-dark__link--hover--Color);
+    --pf-c-nav__link--focus--Color: var(--pf-c-nav--m-dark__link--focus--Color);
+    --pf-c-nav__link--active--Color: var(--pf-c-nav--m-dark__link--active--Color);
+    --pf-c-nav__link--m-current--Color: var(--pf-c-nav--m-dark__link--m-current--Color);
+    --pf-c-nav__link--m-current--BackgroundColor: var(--pf-c-nav--m-dark__link--m-current--BackgroundColor);
+    --pf-c-nav__link--before--BorderColor: var(--pf-c-nav--m-dark__link--before--BorderColor);
+    --pf-c-nav__link--after--BorderColor: var(--pf-c-nav--m-dark__link--after--BorderColor);
+    --pf-c-nav__link--hover--BackgroundColor: var(--pf-c-nav--m-dark__link--hover--BackgroundColor);
+    --pf-c-nav__section-title--Color: var(--pf-c-nav--m-dark__section-title--Color);
+    --pf-c-nav__section-title--BorderBottomColor: var(--pf-c-nav--m-dark__section-title--BorderBottomColor);
+    --pf-c-nav__subnav__link--not--m-current--hover--BorderColor: var(--pf-c-nav--m-dark__subnav__link--not--m-current--hover--BorderColor);
+
+    .pf-c-divider {
+      --pf-c-divider--after--BackgroundColor: var(--pf-c-nav--m-dark__c-divider--BackgroundColor);
+    }
   }
 
   // Disable underline
```

Comment:
Not part of the redesign, but I think we could move this to `.pf-c-nav__link` and be a little more specific with the selector than `[class*="list"] [class*="link"]`